src/main/kotlin/com/example/todo/Service.kt

```diff
@@ -26,9 +28,20 @@ class SimpleTodoService (
         return todoRepository.findAllByStatusOrderByCreatedAtDesc(pageable, todoStatus)
     }
 
-    override fun createTodo(todo: Todo): Todo{
+    override fun createTodo(todo: Todo): Todo {
         val todo = todoRepository.save(todo)
 
         return todo
     }
+
+    override fun toDone(id: Long): Todo?{
+        var todo: Optional<Todo> = todoRepository.findById(id)
+        if (todo.isEmpty){
+            return null
+        }
```

Comment:
P2: Spring Data에 `findByIdOrNull`이라는 함수가 있어요. 다만 null을 반환하는건 앞의 리뷰에서 말씀 드린바와 같이 좋은 패턴이 아니기에 예외를 던지도록 하는게 좋아 보여요

```suggestion
        var todo: Todo? = todoRepository.findByIdOrNull(id)
```

Reply:
https://github.com/jimin0127/kotlin-spring-todo/pull/3/files#diff-5c95af22a165dc67cc793994fa8df97cf84af53fdcc5d3a66810152c81f62be8R35-R38
반영했습니다